frontend/src/lib/components/filters.svelte

```diff
@@ -160,8 +176,9 @@
             class="btn btn-xs {filterAmount ? 'btn-primary' : 'btn-disabled'}"
             on:click={() => {
               $filters.minImdb = 0
-              $filters.tvChecked = true
-              $filters.movieChecked = true
+              for (const key in $filters.checked) {
+                $filters.checked[key] = false
+              }
```

Comment:
We might have talked about it, but I feel like this should work

```suggestion
              for (let key of $filters.checked) {
                key = false
              }
```

Reply:
Didn't work 🤷